File: app/src/main/java/com/cyberIyke/allvideodowloader/webservice/RequestApi.java
package com.cyberIyke.allvideodowloader.webservice;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RequestApi {
    @GET("sg/?")
    Call<SearchModel> getSearchResult(@Query("output") String str, @Query("nresults") int i, @Query("command") String str2);
}

File: app/src/main/java/com/cyberIyke/allvideodowloader/webservice/RequestApi.kt
package com.cyberIyke.allvideodowloader.webservice

import com.cyberIyke.allvideodowloader.webservice.SearchModel
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface RequestApi {
    @GET("sg/?")
    fun getSearchResult(
        @Query("output") str: String?,
        @Query("nresults") i: Int,
        @Query("command") str2: String?
    ): Call<SearchModel?>?
}